hail/src/main/scala/is/hail/types/physical/stypes/concrete/SCanonicalCall.scala

```diff
@@ -91,6 +93,46 @@ class SCanonicalCallSettable(val call: Settable[Int]) extends SCallValue with SS
           cb.append(Code._fatal[Unit](const("invalid ploidy: ").concat(p.toS)))))
     })
   }
+
+  def lgtToGT(cb: EmitCodeBuilder, localAlleles: SIndexableValue, errorID: Value[Int]): SCallCode = {
```

Comment:
lgt_to_gt is a function that translates LGT (local gt) to GT (absolute gt) using the local alleles array. 

For instance, with LGT=0/2 and LA=[0,3,4], the GT is 0/4.